audio/context.rs

```diff
@@ -156,11 +156,14 @@ impl AudioContext {
             })
             .expect("Failed to spawn AudioRenderThread");
 
-        let thread_result: Result<(), AudioSinkError> = result_receiver.recv().unwrap();
+        let thread_result = result_receiver.try_recv();
```

Comment:
This won't work quite right. The recv call is necessary because we need to block until we receive either a success or fail signal on the channel; if we're tiing out in tests, it means that we need to find a way to split the work of setting up the audio backend so that we can signal a success before proceeding with running it synchronously.

Reply:
okey got it, I will revert last commit and try to change how we do the setup.

Reply:
@jdm should we merge this one?

Reply:
I don't think we can merge this without fixing the timeout issue you found, and I think the fix needs to be in this repository.